.drone.yml

```diff
@@ -59,3 +59,185 @@ volumes:
   - name: docker
     host:
       path: /var/run/docker.sock
+---
+kind: pipeline
+name: arm64
+
+platform:
+  os: linux
+  arch: arm64
+
+steps:
+  - name: build
+    image: rancher/dapper:v0.5.8
+    commands:
+      - dapper ci
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+
+  - name: docker-publish-master
+    image: plugins/docker
+    settings:
+      build_args:
+        - ARCH=arm64
+        - VERSION=${DRONE_BRANCH}-${DRONE_COMMIT_SHA:0:8}-head
+      dockerfile: package/Dockerfile
+      password:
+        from_secret: docker_password
+      repo: "rancher/support-bundle-kit"
+      tag: "${DRONE_BRANCH}-head-arm64"
+      username:
+        from_secret: docker_username
+    when:
+      ref:
+        include:
+          - "refs/heads/master"
+          - "refs/heads/release/v*"
+      event:
+        - push
+
+  - name: docker-publish
+    image: plugins/docker
+    settings:
+      dockerfile: package/Dockerfile
+      password:
+        from_secret: docker_password
+      repo: "rancher/support-bundle-kit"
+      tag: "${DRONE_TAG}-arm64"
+      username:
+        from_secret: docker_username
+    when:
+      instance:
+        - drone-publish.rancher.io
+      ref:
+        - refs/head/master
```

Comment:
Why need master branch? Please check other arches as well.

Reply:
This part is directly copied from amd64. Shall we remove this from all arches?

Reply:
If it's not right, then we should remove it for sure.